federation/pkg/kubefed/init/init.go

```diff
@@ -230,10 +230,10 @@ func initFederation(cmdOut io.Writer, config util.AdminConfig, cmd *cobra.Comman
 
 	if !dryRun {
 		return printSuccess(cmdOut, ips, hostnames)
-	} else {
-		_, err := fmt.Fprintf(cmdOut, "Federation control plane runs (dry run)\n")
-		return err
 	}
+	_, err = fmt.Fprintf(cmdOut, "Federation control plane runs (dry run)\n")
+	return err
+
```

Comment:
Unnecessary blank line. It is fine for now. But please take care of them from next time.